Products/ZenPackAdapter/monkey.py

```diff
@@ -23,6 +23,7 @@
 from Products.ZenUtils.CmdBase import CmdBase
 from Products.ZenPackAdapter.cloudpublisher import CloudMetricPublisher
 
+import zenwrapt
```

Comment:
doesn't appear to be used

Reply:
zenwrapt auto-initialized when imported, so nothing more needed to be done. Originally, that code was included in monkey.py, but was separated out for readability and maintainability.

I update zenwrapt.py to have an initialize() method that is now called here and should be more intuitive.